src/context/context.js

```diff
@@ -0,0 +1,10 @@
+import { AuthProvider } from "./auth-context";
+export { useAuth } from "./auth-context";
```

Comment:
Didn't understand use of this statement for this file. You can remove it if not needed

Reply:
to use in various components.. i'll export all the context from this single file.